### DISCUSSION TITLE
Working with Hubspot's hubl syntax in PostCSS

### DISCUSSION THREAD

**Discussion by jazzyclimber** [postcss/postcss#1637] [Q&A]
Hey all -- im building a lot of sites on hubspot these days and trying to incorporate postcss/webpack into the process. 

Hubspot has a jinja based syntax called hubl that is allowed inside of CSS files.

I seem to be unable to find a way to escape/ignore hubl expressions in the postcss compile process. Is this even possible? 

```
{% if statement %}

.selector {
...
}
{% endif %}
```

Is technically valid css. 

Here is a link to the plugin that im writing to try to overcome this issue. Any thoughts/help would be greatly appreciated!

https://github.com/jazzyclimber/postcss-hubl-ignore

**Comment by ai** [comment]
You need a special parser for that :(

you can try to create some solution with `postcss-safe-parser`, but it will be unstable

Creating a new parser is not hard https://github.com/postcss/postcss/blob/main/docs/syntax.md

But I am not sure that it worth it
